bio_pieces/parallel_blast.py

```diff
@@ -73,23 +71,39 @@ def parallel_blast(inputfile, outfile, ninst, db, blasttype, task, blastoptions)
         None if blastx/blastp
     :param str blastoptions: other options to pass to blast
     '''
+    if has_duplicate_args(blastoptions, STATIC_BLAST_ARGS):
```

Comment:
`set(a).intersection(b)` or `filter(a.__contains__, b)`
using `has_duplicate_args` forces me to look for that function definition elsewhere